Source/ACE.Database/DatabaseManager.cs

```diff
@@ -15,10 +15,13 @@ public static class DatabaseManager
 
         public static SerializedShardDatabase Shard { get; private set; }
 
+        public static ServerPropertyDatabase ServerConfig { get; private set; } = new ServerPropertyDatabase();
```

Comment:
I don't think this is needed anymore now that the config has been rolled into the Shard database.